### PR TITLE
Add Ids to PassErrorScope

### PR DESCRIPTION
**Pull request by scoopr**
Also makes it pub, and the places it is used.
Tweak some error messages.

### CODE REVIEW

File: wgpu-core/src/command/bundle.rs [hunk 1]
```diff
@@ -745,7 +745,7 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                         num_dynamic_offsets,
                         bind_group_id,
                     } => {
-                        let scope = PassErrorScope::SetBindGroup;
+                        let scope = PassErrorScope::SetBindGroup(bind_group_id);
```
Comment (kvark) [inline wgpu-core/src/command/bundle.rs:748]: if we go this route, wouldn't it be easier to just have the full `command` cloned into the error?

File: wgpu-core/src/command/bundle.rs [hunk 2]
```diff
@@ -680,7 +680,7 @@ where
 #[derive(Clone, Debug, Error)]
 #[error("{scope}")]
 pub struct RenderBundleError {
-    scope: PassErrorScope,
+    pub scope: PassErrorScope,
```
Comment (kvark) [inline wgpu-core/src/command/bundle.rs:683]: why does it need to be public?

File: wgpu-core/src/command/bundle.rs (hunk 1 above)
Comment (scoopr) [reply wgpu-core/src/command/bundle.rs:748]: Maybe. I don’t currently see a reason though, what else useful information would be had, that isn’t then also displayed in the more specific error?

I don’t see the implementation to be any simpler the either.

File: wgpu-core/src/command/bundle.rs (hunk 2 above)
Comment (scoopr) [reply wgpu-core/src/command/bundle.rs:683]: I need to access it from the wgpu-rs side, as it is information kind of in the side, with the Ids that need to be resolved

File: wgpu-core/src/command/bundle.rs (hunk 1 above)
Comment (kvark) [reply wgpu-core/src/command/bundle.rs:748]: The main benefit is not having a separate `PassErrorScope` enum at all. If you don't think that's a big one, I'm fine with proceeding with your plan :)

**Review by kvark — APPROVED** [review]
bors r+

**Pull request merged**